### DISCUSSION TITLE
[Feature Request] Pdf Export

### DISCUSSION THREAD

**Discussion by kadirkatirci** [vslinko/obsidian-zoom#100] [Ideas]
Hi, thank you for plugin.

when i zoom into heading and try to get pdf export it gives me entire document content. if it is possible to make limited pdf content with the content heading zoomed in it would be very nice.

Thanks.

**Comment by rjst** [comment]
My need for this also brought me here. Especially in daily notes, sometimes I want to focus on a single part, and then export that, but when exporting the view to PDF, the entire document gets exported.

This seems a bit related to the fact that the export to PDF feature doesn't allow exporting just selected text.

Thanks for any help in getting around this
